Swiftfin/Components/ChevronInputButton.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import SwiftUI
+
+struct ChevronInputButton<Value>: View where Value: LosslessStringConvertible & Equatable {
+
+    @Binding
+    private var value: Value
+    @State
+    private var temporaryInputValue: String
+    @State
+    private var isSelected = false
+
+    private let title: String
+    private let description: String?
+    private let subtitle: String
+    private let helpText: String?
+    private let keyboardType: UIKeyboardType
+
+    init(
+        title: String,
+        subtitle: String,
+        description: String? = nil,
+        helpText: String? = nil,
+        value: Binding<Value>,
+        keyboard: UIKeyboardType = .default
+    ) {
+        self.title = title
+        self.subtitle = subtitle
+        self.description = description
+        self.helpText = helpText
+        self._value = value
+        self._temporaryInputValue = State(initialValue: value.wrappedValue.description)
+        self.keyboardType = keyboard
+    }
+
+    // MARK: - Body
+
+    // TODO: Likely want to redo this but better. Needed in
```

Comment:
Needed in? Also how can it be done better in your eyes? A lot of TODOs in this project don't provide enough direction for contributors to help out.

Reply:
TODO: Finish TODO...

Sorry about that, I meant to pull the locations that call this button.

This is called from the TimeIntervalSection of the AddScheduledTaskTriggersView and the HomeSection of the CustomizeViewSettingsView. So, it's "Needed in..." those locations.

I detail my potential issues with this and another comment on this PR. My goal with this button is we now have two locations where we are using an alert to host a TextField used to populate a binding. My hope was to consolidate that work into a single view so we're not redoing the alert logic in multiple places. My thinking is that if we can create one good instance on a button and then use that button everywhere it's easier to maintain.

One I think might be issues or something I could do better:

- I think I need more ways to Init this? That is, do I include the text field in the alert or should I make them so if you can be passed in trailing so you could pass in a textField or a picker or... whatever is needed. Frankly, a lot of what I've done in Swift has been single function so creating something like this with a bit more of a broad potential usage, I want to make sure it's flexible.
- UI wise, do we want to use the Chevron button to display the value? Is there a better way to display this information?
- Lastly, I like the idea of consolidating all of the alert inputs into a single button, but I want to make sure that it's not too broad to begin with. 

Let me know if any of that doesn't make sense! I'm still newer to Swift/SwiftUI so advice is always appreciated.